terminalfighter.py

```diff
@@ -99,11 +109,38 @@ def terminalfighter(screen):
         else:
             print("WARNING: Cannot keep up with 30FPS update time!")
 
-if __name__ == '__main__':
-    GAME_WIDTH = 1000
-    GAME_HEIGHT = 700
+def draw_background():
```

Comment:
So the idea with each weapon having it's own "draw_grid" was that each weapon can determine what the "UI" looked like.

I know for rifle and homing missile we had the same "UI" which was this grid looking thing, but each weapon should be able to decide what the "UI" looks like. So I think you should put these back into the weapons, but call them as you have here, where you call draw_background() once..

Although actually, instead of once, you might have to call draw_background() everytime the player switches the weapon, since it'll draw a new overlay, meaning you'll have to re-draw the black
`pygame.draw.rect(screen, BLACK, pygame.Rect((0, 0), screen.get_size()))`

Also note that we won't always have this being black.. we may have a scrolling background in the future (but maybe it'll update at like 5fps or something) so idk... but i'll be ok for this PR to just have the backgrounds support black only, but I do think we should put this grid stuff back to the weapons so they can choose their own UI

Reply:
I've moved the draw_grid back into the weapons. For now, I've kept the draw background in the global terminalfighter.py though